site/source/tutorials/003_creating_widgets/index.md

```diff
@@ -206,7 +205,7 @@ We have succeeded in rendering the widget, but there seem to be some styling iss
 export default class Worker extends WidgetBase<WorkerProperties>
 ```
 
-The `WorkerProperties` interface adds two new optional properties that we'll be able to use. Now that these are available, let's use them to make the name of the worker dynamic.
+The `WorkerProperties` interface adds two new optional properties that we'll be able to use. Now that these are available, let's use them to make the name of the worker controlled.
```

Comment:
Does someone know what controlled means at this point? If not, maybe an aside or a link?

Reply:
I can put "by the parent widget"

Reply:
ok, that seems reasonable.